.gitignore

```diff
@@ -17,3 +17,6 @@ figwheel_server.log
 *.iml
 .clj-kondo
 .lsp
+node_modules/*
+package.json
```

Comment:
May not be a bad idea to check both the package*.json files into gh instead for stability. But won't block on it.

Reply:
Seems reasonable:
https://github.com/cerner/clara-rules/pull/491/commits/4d0e1dcfaf040da31d4d5c99f63300c2da225412